ex_feels/lib/ex_feels_web/twitter_api/worker.ex

```diff
@@ -3,48 +3,114 @@ defmodule ExFeelsWeb.TwitterApi.Worker do
 
   use GenServer
 
+  require Logger
+
+  alias ExFeelsWeb.TwitterApi.Search
+
   @search_interval 1 * 1000 * 60 * 60 # one hour
   # @search_interval 1 * 1000 * 60 # one minute
 
+  @py_feels [:binary_classifier, :time_series]
+
   def start_link() do
-    GenServer.start_link(__MODULE__, %{})
+    module = __MODULE__
+    state = {"bitcoin", [months: -1]}
```

Comment:
This one line can be changed to perform different searches. Below are some other examples...
```
state = {"dogecoin", [years: -2]}
...
state = {"ethereum", [weeks: -1]}
...
state = {"neo", [days: -1]}
```
You'll have to restart the server after a change since this is compile-time data.